tmt/steps/provision/podman.py

```diff
@@ -364,10 +364,13 @@ def push(
         if not self.is_ready:
             return
 
-        self.debug("Update selinux context of the run workdir.", level=3)
+        assert self.parent.plan.my_run is not None  # narrow type
         assert self.parent.plan.workdir is not None  # narrow type
+
         # Relabel workdir to container_file_t if SELinux supported
-        if tmt.utils.is_selinux_supported():
+        self.debug("Update selinux context of the run workdir.", level=3)
+
+        if self.parent.plan.my_run.runner.facts.has_selinux:
```

Comment:
What about adding at least a short docstring to https://tmt.readthedocs.io/en/stable/code/autodocs/tmt.html#tmt.base.Run.runner ?

Reply:
+1, could be visible there

Reply:
fine with me, @psss ?

Reply:
@psss ping?

Reply:
Yeah, a separate pull request sounds good.